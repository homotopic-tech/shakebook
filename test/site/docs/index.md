---
title: The First Doc
description: The First Doc
---

Lorem ipsum
