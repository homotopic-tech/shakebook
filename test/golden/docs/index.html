<html>
  <head>
    <title>The First Doc</title>
  </head>
  <body>
    <h1>This post page is called The First Doc</h1>
    <p>It was generated from the source docs/index.md</p>
    <p>For the url /docs/index.html</p>
    <p><p>Lorem ipsum</p></p>
    <p>Here are the subsections:
      <ul>
        <li>
          <a href="/docs/1/index.html">The First Subsection</a>
          <p>The First Subsection</p>
        </li>
        <li>
          <a href="/docs/2/index.html">The Second Subsection</a>
          <p>The Second Subsection</p>
        </li>
      </ul>
    </p>
  </body>
</html>
